src/modules/shared/components/ContactIconButton/ContactIconButton.tsx

```diff
@@ -0,0 +1,29 @@
+import clsx from 'clsx';
+import React, { FC, MouseEventHandler } from 'react';
+
+import { Icon } from '..';
+import { Button } from '../Button/Button.stories';
+
+import styles from './ContactIconButton.module.scss';
+
+export interface ContactIconButtonProps {
+	color?: string;
+	onClick?: MouseEventHandler<HTMLButtonElement>;
+}
+
+const ContactIconButton: FC<ContactIconButtonProps> = ({ color, onClick }) => {
```

Comment:
Waar verschilt deze knop van een icon button?

Reply:
Niet veel, puur door context , lichte styling in `ContactIconButton.module.scss` en herbruikbaarheid doorheen de applicatie (zie exacte herhaling in dit [scherm](https://xd.adobe.com/view/86b0be3f-d968-4a98-7784-d5d00cd42f98-f081/screen/99438b88-fe36-4790-96ad-e5a51a6b9fa9/specs/))